### ISSUE TITLE
Validation test set too small

### ISSUE THREAD

**Issue by Jayanth-Balasubramanian** [Jayanth-Balasubramanian/Pneumonia-Detection#1]
### Problem: 
The existing validation set has only 8 images per class, so the validation accuracy scores are not very reliable. 

### Solution idea:
Split the training dataset 85-15 and use the smaller part for validation.
